performance/src/main/java/org/hibernate/validator/performance/cascaded/CascadedValidation.java

```diff
@@ -65,20 +66,24 @@ public void testCascadedValidation(CascadedValidationState state) {
 
 		Set<ConstraintViolation<Person>> violations = state.validator.validate( kermit );
 		assertThat( violations ).hasSize( 0 );
+
+		bh.consume( violations );
 	}
 
+/*
```

Comment:
Hi John,

You commented this, is this what you wanted? BTW, the use of the constant looks weird as it's not made to be used here (it is used in the for loops).

But there's definitely something weird with the constants as we don't use them consistently.